tests/backup/backup_test.go

```diff
@@ -237,6 +237,158 @@ var _ = Describe("{CreateMultipleUsersAndGroups}", func() {
 
 })
 
+// Validate that user can't duplicate a shared backup without registering the cluster
+var _ = Describe("{DuplicateSharedBackup}", func() {
+	userName := "testuser1"
+	firstName := "firstName"
+	lastName := "lastName"
+	email := "testuser10@cnbu.com"
+	password := "Password1"
+	numberOfBackups := 1
+	var backupName string
+	userContexts := make([]context.Context, 0)
+	var contexts []*scheduler.Context
+	var bucketName string
+	var BackupLocationName string
+	var backupLocationUID string
+	var cloudCredUID string
+	var CloudCredUidList []string
+	var appContexts []*scheduler.Context
+	var bkpNamespaces []string
+	var clusterUid string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	var credName string
+	bkpNamespaces = make([]string, 0)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("DuplicateSharedBackup",
+			"Share backup with user and duplicate it", nil, 82937)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("Validate shared backup is not duplicated without cluster", func() {
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		providers := getProviders()
+		buckets := getBucketName()
+		backupName = fmt.Sprintf("%s-%v", BackupNamePrefix, time.Now().Unix())
+		Step("Validate applications", func() {
+			log.InfoD("Validate applications")
+			ValidateApplications(contexts)
+		})
+		Step("Create User", func() {
+			err = backup.AddUser(userName, firstName, lastName, email, password)
+			dash.VerifyFatal(err, nil, "Verifying user creation")
+
+		})
+		Step("Adding Credentials and Registering Backup Location", func() {
+			log.InfoD("Creating cloud credentials and backup location")
+			for _, provider := range providers {
+				bucketName = fmt.Sprintf("%s-%v", provider, buckets[0])
+				cloudCredUID = uuid.New()
+				CloudCredUidList = append(CloudCredUidList, cloudCredUID)
+				backupLocationUID = uuid.New()
+				credName = fmt.Sprintf("autogenerated-cred-%v", time.Now().Unix())
+				CreateCloudCredential(provider, credName, cloudCredUID, orgID)
+				log.InfoD("Created Cloud Credentials with name - %s", credName)
+				time.Sleep(time.Minute * 1)
+				BackupLocationName = fmt.Sprintf("autogenerated-backup-location-%v", time.Now().Unix())
+				CreateBackupLocation(provider, BackupLocationName, backupLocationUID, credName, cloudCredUID, bucketName, orgID, "")
```

Comment:
I think @snigdha-px  has made some changes to this method where it returns. If so handle it

Reply:
@snigdha-px's PR isn't approved yet